src/controllers/logging.js

```diff
@@ -0,0 +1,11 @@
+import Logs from '../collections/logs';
+import errorMessage from '../utils/error';
+
+export default function(req, res) {
+    new Logs().fetch().then((log) => {
```

Comment:
this can just be `Logs.fetch()` rather than `new Logs().fetch()`. minor, but, idk, meh